.travis.yml

```diff
@@ -102,9 +104,16 @@ after_success:
               git_desc="$(git describe)";
               curl --ftp-create-dirs -T KVIrc.dmg -u $FTP_USER:$FTP_PASS ${FTP_HOST}KVIrc-${kvi_version}-dev-${git_desc}.dmg
           fi
+    - |
+          if [ ! -z "$DEBUILD" ] && [ "$TRAVIS_REPO_SLUG" == "kvirc/KVIrc" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" == "master" ]; then
+              cd dist
+              bash build_debian.sh -p || exit 1
+          fi
 env:
   global:
-  # These 3 lines are: $FTP_USER, $FTP_PASS, $FTP_HOST
+  # These 5 lines are: $FTP_USER, $FTP_PASS, $FTP_HOST, $GPGPASS, $AESPASS
```

Comment:
Let's keep these comments separate, to know which lines should be updated when necessary